src/appleseed/renderer/kernel/rendering/defaultrenderercontroller.h

```diff
@@ -59,6 +59,12 @@ class APPLESEED_DLLSYMBOL DefaultRendererController
     // This method is called after rendering was aborted.
     void on_rendering_abort() override;
 
+    // This method is called after rendering was paused.
+    void on_rendering_paused() override;
```

Comment:
For consistency with the other slots, should be `on_rendering_pause()` and `on_rendering_resume()`.